src/components/Sections/SectionView.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react'
+import cx from 'classnames'
+
+import SquareAppIcon from 'cozy-ui/transpiled/react/SquareAppIcon'
+import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
+
+import flags from 'components/Sections/flags.json' // TODO: to be fetched from cozy-flags
+import {
+  DisplayMode,
+  Section,
+  SectionViewProps
+} from 'components/Sections/SectionsTypes'
+import { SectionHeader } from 'components/Sections/SectionHeader'
+
+const useSectionDisplayMode = (
+  section?: Section
+): [DisplayMode, React.Dispatch<React.SetStateAction<DisplayMode>>] => {
+  const { isMobile } = useBreakpoints()
+  const initialDisplayMode = section?.layout?.[isMobile ? 'mobile' : 'desktop']
```

Comment:
Perhaps a boolean state would be enough, included in `SectionView` directly without needed an hook has it never used anywhere else